Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.Tests/Lib/Helpers.cs

```diff
@@ -169,5 +140,115 @@ internal static void SetTestFileLastModified(string path, DateTime date)
+            {
+                fingerprint = BitConverter.ToString(e.FingerPrint).Replace("-", ":");
+                e.CanTrust = true;
+            };
+            client.Connect();
+            client.Disconnect();
+        }
+
+        return fingerprint;
+    }
+
+    internal static string GetServerFingerprintAsSHA256Base64String()
+    {
+        var fingerprint = "";
+        using (var client = new SftpClient(_dockerAddress, 2222, _dockerUsername, _dockerPassword))
+        {
+            client.HostKeyReceived += delegate (object sender, HostKeyEventArgs e)
+            {
+                using (SHA256 mySHA256 = SHA256.Create())
+                {
+                    fingerprint = Convert.ToBase64String(mySHA256.ComputeHash(e.HostKey));
+                }
+                e.CanTrust = true;
+            };
+            client.Connect();
+            client.Disconnect();
+        }
+
+        return fingerprint;
+    }
+
+    internal static string GetServerFingerprintAsSHA256HexString()
+    {
+        var fingerprint = "";
+        using (var client = new SftpClient(_dockerAddress, 2222, _dockerUsername, _dockerPassword))
+        {
+            client.HostKeyReceived += delegate (object sender, HostKeyEventArgs e)
+            {
+                using (SHA256 mySHA256 = SHA256.Create())
+                {
+                    fingerprint = ToHex(mySHA256.ComputeHash(e.HostKey));
+                }
+                e.CanTrust = true;
+            };
+            client.Connect();
+            client.Disconnect();
+        }
+
+        return fingerprint;
+    }
+
+    internal static string GetServerFingerprintAsSHA1String()
+    {
+        var fingerprint = "";
+        using (var client = new SftpClient(_dockerAddress, 2222, _dockerUsername, _dockerPassword))
+        {
+            client.HostKeyReceived += delegate (object sender, HostKeyEventArgs e)
+            {
+                using (var sha1 = SHA1.Create())
+                {
+                    var hash = sha1.ComputeHash(e.HostKey);
+                    fingerprint = string.Concat(hash.Select(b => b.ToString("x2")));
+                }
+                e.CanTrust = true;
+            };
+            client.Connect();
+            client.Disconnect();
+        }
+
+        return fingerprint;
+    }
+
+    internal static string HexStringToB64String(string input)
+    {
+        return Convert.ToBase64String(ConvertHexStringToHex(input));
+    }
+
+    internal static byte[] ConvertHexStringToHex(string hex)
+    {
+        var arr = new byte[hex.Length / 2];
+        for (var i = 0; i < arr.Length; i++)
+        {
+            arr[i] = Convert.ToByte(hex.Substring(i * 2, 2), 16);
+        }
+        return arr;
+    }
+
+    internal static string ConvertHexStringToBase64(string hexString)
+    {
+        Func<char, int> parseNybble = c => (c >= '0' && c <= '9') ? c - '0' : char.ToLower(c) - 'a' + 10;
```

Comment:
Do you mean "nibble"?

Reply:
I removed this method altogether because it was not used.